serval-agent/src/api/storage.rs

```diff
@@ -15,11 +15,7 @@ pub async fn get_blob(
 ) -> impl IntoResponse {
     // Yeah, I don't like this.
     let state = state.lock().await;
-
-    let Some(storage) = state.storage.as_ref() else {
-        // todo: in this case, we should proxy this request to another node that is advertising the serval_storage role
-        return (StatusCode::SERVICE_UNAVAILABLE, "Storage not available").into_response();
-    };
+    let storage = state.storage.as_ref().unwrap();
```

Comment:
“I'll just put this over here with the rest of the don't-like-this.”

![the-it-crowd-moss](https://user-images.githubusercontent.com/41654/212148902-aa576efb-9041-4257-8dc2-8c176f091911.gif)